iree/compiler/Conversion/Common/BUILD

```diff
@@ -55,6 +55,6 @@ cc_library(
         "@llvm-project//mlir:StandardOps",
         "@llvm-project//mlir:Transforms",
         "@llvm-project//mlir:VectorOps",
-        "@mlir-hlo//:hlo",
+        "@org_tensorflow//tensorflow/compiler/mlir/hlo:hlo",
```

Comment:
Since the main perfect does not depend on tensorflow, this was intentional. Why did you find that you had to change it?